src/attributes.jl

```diff
@@ -440,6 +440,15 @@ given.
 """
 struct Silent <: AbstractOptimizerAttribute end
 
+"""
+    TimeLimit
+
+An optimizer attribute for setting a time limit to an optimization. When `set`
+to `0`, it deactivate the solver time limit. The default value is `0`. The time
```

Comment:
+1 for `nothing` instead of `0` as a default.

Reply:
Why not set the default to whatever default the solver has. Some solver have a default time limit

Reply:
No solvers should have a default finite time limit. They might represent "no limit" it as either a large number or as 0.0, but we should abstract over that.

Reply:
Ok so we should say that if a solver has a default time limit, it should be disabled